### DISCUSSION TITLE
Discord: Removing the generally unused roles system

### DISCUSSION THREAD

**Discussion by JoshuaKGoldberg** [typescript-eslint/typescript-eslint#9962] [Community Feedback]
Back in https://github.com/typescript-eslint/typescript-eslint/discussions/7103, we mentioned rolling out some kind of Discord-oriented system for recognizing contributors. https://typescript-eslint.io/maintenance/contributor-tiers/#advancement mentions posting in the `#roles` channel on our Discord.

But:

* Roughly nobody has used the system
* We don't have anything to tie the system to (e.g. rewards, community shoutouts, etc.)
* We seem to be doing fine without it

If nobody objects, I'd like to remove the `#roles` channel and mention in https://typescript-eslint.io/maintenance/contributor-tiers/#advancement.

Posting here for discussion per https://typescript-eslint.io/maintenance#changes-to-governance. Does anybody have any objections? 🙂

**Comment by NanderTGA** [comment]
This is the first time I've heard of this system. I just noticed I got a contributor role at some point after having fixed a few links and thought ok cool. Isn't that the main point of having the roles?

We have a somewhat similar system in another Discord server about a random game. When you hit a certain level, you can request a role. Is it useful? No, not really, apart from one channel only lvl 100+ players can talk in.
But roles are nice and somewhat useful so people know who they are dealing with and how much they may know about the project and the community surrounding it. A big use for roles is just having and giving them to people, for recognition (like the docs say).

From what I see, the only effort required is giving people the role when they ask for it. I don't think anyone will care whether you keep this system or not so if you're wasting time, sure, get rid of it.
But if you're not wasting time on it, why not keep it?

**Reply by JoshuaKGoldberg** [reply]
😂 I'd forgotten about the roles system myself until now... but someone just picked up on it and I don't think we'll want to keep spending bandwidth + docs space to keep it up.
